index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homyfood</title>
+    <meta
+      name="description"
+      content="ohmyfood, reservez le menu qui vous convient"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/normalize.css" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
+  <body>
+    <header>
+      <img
+        src="./assets/images/logo/ohmyfood.png"
+        alt="Logo du site Homyfood"
+      />
+    </header>
+    <main>
+      <div id="localisation">
+        <i></i>
+        <label for="search"><i class="fa-solid fa-location-dot"></i></label>
+        <input
+          type="search"
+          id="search"
+          name="Ville"
+          placeholder="Paris, Belleville"
+        />
+      </div>
+      <section id="exploration">
+        <h4>Réservez le menu qui vous convient</h4>
+        <p>
+          Découvrez des restaurants d'exception, séléctionnés par nos soins.
+        </p>
+        <a href="#"><button>Explorer nos restaurants</button></a>
+      </section>
+      <section id="working">
+        <h4>Fonctionnement</h4>
+        <ul>
+          <li>
+            <button>
+              <a href="#"
+                ><i class="fa-solid fa-mobile-screen-button"></i>Choisissez un
+                réstaurant</a
+              >
+            </button>
+          </li>
+          <li>
+            <button>
+              <a href="#"
+                ><i class="fa-solid fa-ellipsis-vertical"></i
+                ><i class="fa-solid fa-bars"></i>Composez votre menu</a
+              >
+            </button>
+          </li>
+          <li>
+            <button>
+              <a href="#"
+                ><i class="fa-solid fa-store"></i>Dégustez au restaurant</a
+              >
+            </button>
+          </li>
+        </ul>
+      </section>
+      <section id="restaurants">
+        <h4>restaurants</h4>
+        <article>
+          <a href="#">
+            <figure>
+              <img
+                src="./assets/images/restaurants/jay-wennington-N_Y88TWmGwA-unsplash.jpg"
+                alt="Restaurant La note enchantée"
+              />
+              <figcaption>
+                <h3>La palette du goût</h3>
+                <p>Ménilmontant</p>
+                <i class="fa-regular fa-heart"></i>
+              </figcaption>
+            </figure>
+          </a>
```

Comment:
You can make a hred for all because else we can click on heart (if we do it, we click on menu). So have a href for img then for h3 and p in figatpion)